uaclient/cli/formatter.py

```diff
@@ -133,9 +130,15 @@ def _get_column_sizes(self) -> List[int]:
         return column_sizes
 
     def __str__(self) -> str:
+        return self.to_string()
+
+    def to_string(self, length: Optional[int] = None) -> str:
```

Comment:
can we rename this variable to `line_lenght` ? Just to make the meaning of this variable more explicit

Reply:
done